### PR TITLE
Add the comment file option to regression job

### PR DESCRIPTION
**Pull request by deepshikhaaa**

### CODE REVIEW

File: build-gluster-org/scripts/centos-regression.sh [hunk 1]
```diff
@@ -117,20 +117,14 @@ else
...
-    V="+1"
-    VERDICT="SUCCESS"
-else
-    V="-1"
-    VERDICT="FAILED"
-fi
-
 if [ ${RET} -ne 0 ]; then
+    # Update Gerrit with the success/failure status
+    sudo mv /tmp/gluster_regression.txt $WORKSPACE || true
```
Comment (mscherer) [inline build-gluster-org/scripts/centos-regression.sh:122]: Why || true ? It if fail, we want to see it, no ?

Comment (mscherer) [reply build-gluster-org/scripts/centos-regression.sh:122]: Ok, didn't see it was here before.

File: build-gluster-org/scripts/centos-regression.sh [hunk 2]
```diff
@@ -117,20 +117,14 @@ else
...
     echo "Logs are archived at Build artifacts: https://build.gluster.org/job/${JOB_NAME}/${BUILD_ID}"
+else
+    # leave result file empty if there is no failure, in context of `comment-file` in ghprb plugin
+    touch gluster_regression.txt
 fi
 
-# Update Gerrit with the success/failure status
-sudo mv /tmp/gluster_regression.txt $WORKSPACE || true
-sudo chown jenkins:jenkins gluster_regression.txt || true
-vote_gerrit "$V" "$VERDICT" gluster_regression.txt
```
Comment (mscherer) [inline build-gluster-org/scripts/centos-regression.sh:135]: We should also remove the function if we do not use it, no ?

File: build-gluster-org/scripts/centos-regression.sh (hunk 1 above)
Comment (deepshikhaaa) [reply build-gluster-org/scripts/centos-regression.sh:122]: It would fail when the test suite passes as that time the file will not be created.

**Pull request merged**